tensorflow/core/kernels/pooling_ops_common.h

```diff
@@ -68,6 +68,69 @@ struct PoolParameters {
   TensorFormat data_format;
 };
 
+#ifdef TENSORFLOW_USE_SYCL
```

Comment:
That will cause conflicts.. can you make PR for SYCL2DPoolParams on it's own? So we can merge that in ahead of the other changes?